try {
console.log(program )
} 
catch (error) {
    console.log("this is a error")
    console.log(p)
}

finally{
    console.log("i am good boy")
}